packages/web-components/src/components/quote/quote.ts

```diff
@@ -89,47 +96,66 @@ class C4DQuote extends StableSelectorMixin(LitElement) {
     switch (this.markType) {
       case QUOTE_TYPES.SINGLE_CURVED:
         return html`
-          <span class="${prefix}--quote__mark">‘</span>
-          <blockquote class="${prefix}--quote__copy">
-            <slot></slot><span class="${prefix}--quote__mark-closing">’</span>
+          <span class="${prefix}--quote__mark" part="mark">‘</span>
+          <blockquote class="${prefix}--quote__copy" part="copy">
+            <slot></slot
+            ><span class="${prefix}--quote__mark-closing" part="mark-closing"
+              >’</span
+            >
```

Comment:
I wonder if we should make the naming consistent by doing something like this in all the places we render the quotes:
```suggestion
          <span class="${prefix}--quote__mark" part="mark mark--opening">‘</span>
          <blockquote class="${prefix}--quote__copy" part="copy">
            <slot></slot
            ><span class="${prefix}--quote__mark-closing" part="mark mark--closing"
              >’</span
            >
```
What do you think?